rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,18 @@
  */
 public final class RxView {
   /**
+   * Create an observable of attach and detach events on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult
+  public static Observable<ViewAttachEvent> attachEvents(View view) {
```

Comment:
Does this deserve a pair of siblings?
```java
public static Observable<Object> attaches(View view) { .. }
public static Observable<Object> detaches(View view) { .. }
```
These are zero-allocation operators which appeals to me from an efficiency perspective, but I definitely struggle with the duplication they require on a code-size front (for those not using ProGuard, who I probably shouldn't care about and actually includes myself).